### DISCUSSION TITLE
Onion V4.2.0-RC Swap Triggers Not Working

### DISCUSSION THREAD

**Discussion by courgi** [OnionUI/Onion#1053] [Support] [answered]
Swap Triggers (L<>L2, R<>R2) under **Tweaks<Advanced** do not swap. 
I do see there was a change in RC..
- **Swap L1/R1 and L2/R2 in Package Manager by @schmurtzm in https://github.com/OnionUI/Onion/pull/960**

However, toggle button does not behave as expected and does nothing in RC
Same results when upgrading from previous beta as well as fresh install of RC
Tried Reset Settings which does not resolve 

Tested on Miyoo Mini +Plus

**Comment by Aemiii91** [comment, marked as answer]
The trigger swap in Tweaks is in-game only (and it's working).
